libsolidity/formal/BMC.cpp

```diff
@@ -424,6 +535,28 @@ bool BMC::visit(TryStatement const& _tryStatement)
 	return false;
 }
 
+bool BMC::visit(Break const& breakStatement)
+{
+	(void)breakStatement;
+
+	LoopControl
+		control(LoopControlKind::Break, currentPathConditions(), copyVariableIndices());
+	loopScopes.top().emplace_back(control);
+
+	return false;
+}
+
+bool BMC::visit(Continue const& continueStatement)
```

Comment:
```suggestion
bool BMC::visit(Continue const&)
```
Then you can remove the `(void)...` line.